documentation/modules/ROOT/pages/connectors/sqlserver.adoc

```diff
@@ -18,82 +21,121 @@ Want to help us further hone and improve it? link:/docs/contribute/[Learn how].
-This way, we start with a consistent view of all of the data, yet continue reading without having lost any of the changes made while the snapshot was taking place.
-
-The connector is also tolerant of failures.
-As the connector reads changes and produces events, it records the position in the database log (_LSN / Log Sequence Number_), that is associated with _CDC_ record, with each event.
-If the connector stops for any reason (including communication failures, network problems, or crashes), upon restart it simply continues reading the _CDC_ tables where it last left off.
-This includes snapshots: if the snapshot was not completed when the connector is stopped, upon restart it begins a new snapshot.
+The {prodname} SQL Server connector is based on the https://docs.microsoft.com/en-us/sql/relational-databases/track-changes/about-change-data-capture-sql-server?view=sql-server-2017[change data capture] 
+feature that is available in https://blogs.msdn.microsoft.com/sqlreleaseservices/sql-server-2016-service-pack-1-sp1-released/[SQL Server 2016 Service Pack 1 (SP1) and later] Standard edition or Enterprise edition.
+The SQL Server capture process monitors designated databases and tables, and stores the changes into specifically created _change tables_ that have stored procedure facades.
+
+To enable the {prodname} SQL Server connector to capture change event records for database operations, 
+you must first enable change data capture on the SQL Server database. 
+CDC must be enabled on both the database and on each table that you want to track.
+After you set up CDC on the source database, the connector can capture row-level `INSERT`, `UPDATE`, and `DELETE` operations  
+that occur in the database.
+The connector writes event records for each source table to a Kafka topic especially dedicated to that table.
+One topic exists for each tracked table.
+Client applications read the Kafka topics for the database tables that they follow, and can respond to the row-level events they they consume from those topics.
+
+The first time that the connector connects to a SQL Server database or cluster, it takes a consistent snapshot of the schemas for all tables for which it is configured to capture changes,
+and streams this state to Kafka.
+After the snapshot is complete, the connector continuously captures subsequent row-level changes that occur.
+By first establishing a consistent view of all of the data, the connector can continue reading without having lost any of the changes that were made while the snapshot was taking place.
+
+The {prodname} SQL Server connector is tolerant of failures.
+As the connector reads changes and produces events, it records the position in the database log (_LSN / Log Sequence Number_) for each event.
+If the connector stops for any reason (including communication failures, network problems, or crashes), after a restart the connector resumes reading the SQL Server _CDC_ tables from the last point that it read.
```

Comment:
Worth mentioning that the offset is committed periodically, not after each change event. I.e. there may be some duplicates after a crash.